build.gradle

```diff
@@ -132,6 +136,17 @@ applicationDistribution.into('bin') {
     fileMode = 0755
 }
 
+docker {
+    name 'scalarlabs/cassy'
+    files tasks.distTar.outputs, 'docker/logback.xml', 'docker/entrypoint.sh'
+    copySpec.with {
+        from('.') {
+            include 'scripts/**'
+            into '.'
+        }
+    }
```

Comment:
(IMO) I think it would be nice to add these.
```
pull true
noCache true
```

Reply:
Thanks!
Added `noCache true`,  but `pull true` doesn't seem to be needed because the base image `openjdk:8u212-jre-slim-stretch` is specified with a fixed version in Dockerfile.

Reply:
OK!

I'm not sure how works when the image retagged, So I just thought it would be nice to have `pull true`.